.github/workflows/cd.yml

```diff
@@ -157,6 +157,7 @@ jobs:
           ls -la $BINARY_DEST_PATH
           ls -la $BINARY_DEST_PATH.sha256sum.txt
       - name: Upload Release Assets
+        if: ${{ matrix.name.artifact != 'relay' }}  # Hide relay binary from public releases
```

Comment:
We are sending people to this page to download binaries, and having this here makes the user have to click "Show more" to see binaries they may be looking for.